Libraries/LibWeb/Compression/DecompressionStream.cpp

```diff
@@ -32,8 +32,6 @@ WebIDL::ExceptionOr<GC::Ref<DecompressionStream>> DecompressionStream::construct
     auto decompressor = [&, input_stream = MaybeOwned<Stream> { *input_stream }]() mutable -> ErrorOr<Decompressor> {
         switch (format) {
         case Bindings::CompressionFormat::Deflate:
-            // FIXME: Our zlib decompressor assumes the initial data contains the zlib header. We don't have any data yet,
```

Comment:
There's an analogous FIXME here that should be removed as well (and the test enabled):
https://github.com/LadybirdBrowser/ladybird/blob/5bcba896c2da509d3401d4e08a2e51026b011c77/Tests/LibWeb/Text/input/Compression/DecompressionStream.html#L5-L6